src/bikeracks/templates/scripts.js

```diff
@@ -605,15 +582,13 @@ BikeMap.prototype.arrowHTML = function(state) {
 BikeMap.prototype.popupContent = function(state) {
     let onlineStatus,
         voterStatus;
-
-    onlineStatus = Boolean(this.auth.currentUser);
     
     /* state : {
      *     address: address (string),
      *     latitude: latitude (string),
      *     longitude: longitude (string),
-     *     rack_id: rack_id (string),
-     *     status: string (is it a temp marker or not, can be determined based on whether this rack has a status or not),
+     *     rack_id: rack_id (string), determine if a marker is a temporary marker
+     *         or not by if they have a rack_id or not
      *     user_id: user_id (string, empty? (or undefined) if user is not signed in)
```

Comment:
`empty? (or undefined)` ?  What does that mean here? Will `state` always have this attribute with at least an empty string value?

Reply:
Changed the comment.